CHANGES.rst

```diff
@@ -77,6 +77,9 @@ astropy.units
   and also pickled and unpicked also for ``protocol`` >= 2.
   This does not work for big-endian architecture with older ``numpy<1.21.1``. [#12583]
 
+- Added a ``mass_length_time`` equivalency for converting mass, length, and time
+  based on geometric units.
+
```

Comment:
For Astropy code contributions we actually don't directly commit to CHANGES.rst, but instead use a system call ``towncrier`` to assemble the changelog.
See https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#add-a-changelog-entry
I see you added a changelog fragment already, so this can be removed.

```suggestion
```